### ISSUE TITLE
--split-chapters : FFMPEG does not find the mp3 file to split

### ISSUE THREAD

**Issue by s3n0** [yt-dlp/yt-dlp#7823]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

Hi.

**The issue:** FFMPEG does not find the mp3 file to split... using the `--split-chapters` argument. The mp3 file is successfully downloaded and saved to the Y: drive. The mp3 file represents a complete set of all chapters - together as one whole audio file. 

**Possible cause:** However, the file name does not contain the trailing number, shown in square brackets `[]`, at the end of the file name (as reported by the error message). Probably because of the trailing number in square `[]` brackets ? I believe that this could be a mistake ?

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', '--output', 'Y:\\%(title)s.%(ext)s', '--embed-thumbnail', '--embed-metadata', '--split-chapters', '--extract-audio', '--audio-format', 'mp3', 'https://www.youtube.com/watch?v=3CUDp_kBB5w']
[debug] Encodings: locale cp1250, fs utf-8, pref cp1250, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg n4.3.2-161-g4383b9e8a3, ffprobe n4.3.2-161-g4383b9e8a3
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: 5ff3e702171a50175c34397494e2d18ce35d771c2110b1e59bd173ec2fb352aa
yt-dlp is up to date (stable@2023.07.06)
[youtube] Extracting URL: https://www.youtube.com/watch?v=3CUDp_kBB5w
[youtube] 3CUDp_kBB5w: Downloading webpage
[youtube] 3CUDp_kBB5w: Downloading ios player API JSON
[youtube] 3CUDp_kBB5w: Downloading android player API JSON
[youtube] 3CUDp_kBB5w: Downloading m3u8 information
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, size, br, asr, vext, aext, hasaud, id
[info] 3CUDp_kBB5w: Downloading 1 format(s): 251
[info] Downloading video thumbnail 41 ...
[info] Writing video thumbnail 41 to: Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.webp
[download] Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.mp3 has already been downloaded
[debug] ffmpeg command line: ffprobe -show_streams "file:Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.mp3"
[ExtractAudio] Not converting audio Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.mp3; file is already in target format mp3
[Metadata] Adding metadata to "Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.mp3"
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -i "file:Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.mp3" -i "file:Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.meta" -map 0 -dn -ignore_unknown -c copy -map_metadata 1 -write_id3v1 1 -metadata "title=Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa" -metadata "date=20230812" -metadata "description=Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa July 29, 2023 Molokai Hawaii. -- Lono Music - https://www.lonomusic.com /  https://www.youtube.com/kaupeepee / MOLOKAI IWI BAND https://www.youtube.com/@molokaiiwiband9468 -- Please subscribe to us: http://youtube.com/subscription_center?add_user=photolukehawaii #molokai #hawaiitravel #lonomusic #hawaiilife #hawaii #kaunakakai #virtualdrive #virtualtour #hawaiianmusic #hawaiianculture #photolukehawaii

Menu:
0:00 -- Big Wind Kite Factory in Maunaloa
0:22 — Mauka Kamiano by Lono on Molokai IWI https://lonomusic.com/product/iwi-digital-download/
3:33 — Mauna Alani by Lono on Old Style II https://lonomusic.com/product/old-style-ii-digital-download/
6:35 — Sunshine Slack by Lono Old Style IV https://lonomusic.com/product/old-style-iv-digital-download/
10:12 — E Hele Pohaku by Lono on Old Style III https://lonomusic.com/product/old-style-iii-digital-download/
13:05 — Kila Kila Na Roughrider by Lono on Old Style II https://lonomusic.com/product/old-style-ii-digital-download/
15:38 — Moloka’i T.M.H. by Lono on Old Style IV https://lonomusic.com/product/old-style-iv-digital-download/
18:58 — Saturday Market by Lono on Old Style V https://lonomusic.com/product/old-style-v-digital-download/
23:18 — Botonui by Lono on Old Style III https://lonomusic.com/product/old-style-iii-digital-download/
25:23 -- Pala'au State Park parking lot

Support PhotoLukeHawaii:
— PhotoLukeHawaii on YouTube https://www.youtube.com/photolukehawaii
— PhotoLukeHawaii on Twitch https://www.twitch.tv/photolukehawaii
— Join PhotoLukeHawaii Membership on YouTube for $1.99/month to get access to perks, emotes and a badge https://www.youtube.com/channel/UCAUdpc0sN1CewnyOyqQHXcQ/join
— PhotoLukeHawaii on Patreon for only $3 https://www.patreon.com/photolukehawaii
— PhotoLukeHawaii on PayPal https://www.paypal.me/photolukehawaii
— PhotoLukeHawaii merchandise on Teespring at https://www.teespring.com/stores/photolukehawaii
— PhotoLukeHawaii Travels and Bon Dance: https://www.youtube.com/photolukehawaiitravelsandbondance
Follow us on:
— PhotoLukeHawaii Instagram: https://www.instagram.com/photolukehawaii
— PhotoLukeHawaii Facebook: https://www.facebook.com/Photolukehawaii

For all our Hawaii Videos click here https://www.youtube.com/playlist?list=PLxfXsQszwAJQU5bT5MgrQxhEVuZXjIEAU

— Hawaii Covid Information for all islands https://hawaiicovid19.com

PhotoLukeHawaii is an Amazon Affiliate. These links will take you to Amazon.com to the equipment we use to learn more about them or to buy them.  If you buy it through our link we will get credit for it.  Thank you for your support if you do make a purchase!
Equipment Used:
Final Cut Pro X, Adobe Photoshop, Google Earth, Epidemic Sound
iMac 5K 27”, 3 GHz 6-Core Intel Core i5, 32gb Ram, Radeon Pro 570X 4 GB
GoPro Hero 11: https://gopro.com
GoPro Hero 9: https://amzn.to/2FADMDI
GoPro Volta: https://amzn.to/42iJfE3
Hohem Pro 4: https://amzn.to/3nu8QqQ
Magnetic Swivel Clip: https://amzn.to/3ZY63bv
Head Strap w/ Quick Clip: https://amzn.to/3KFKcQY
Mini Handle Tripod: https://amzn.to/3yP41hL
GoPro Suction Cup Mount:  https://amzn.to/2SfMMPx
Micro SD card: https://amzn.to/31aXRI8
GoPro Hero 9 Battery: https://amzn.to/34XtWVG
GoPro Enduro Battery: https://gopro.com
Hero 9, 10, 11 Dual Battery Charger: https://amzn.to/2MksVRb" -metadata "synopsis=Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa July 29, 2023 Molokai Hawaii. -- Lono Music - https://www.lonomusic.com /  https://www.youtube.com/kaupeepee / MOLOKAI IWI BAND https://www.youtube.com/@molokaiiwiband9468 -- Please subscribe to us: http://youtube.com/subscription_center?add_user=photolukehawaii #molokai #hawaiitravel #lonomusic #hawaiilife #hawaii #kaunakakai #virtualdrive #virtualtour #hawaiianmusic #hawaiianculture #photolukehawaii

Menu:
0:00 -- Big Wind Kite Factory in Maunaloa
0:22 — Mauka Kamiano by Lono on Molokai IWI https://lonomusic.com/product/iwi-digital-download/
3:33 — Mauna Alani by Lono on Old Style II https://lonomusic.com/product/old-style-ii-digital-download/
6:35 — Sunshine Slack by Lono Old Style IV https://lonomusic.com/product/old-style-iv-digital-download/
10:12 — E Hele Pohaku by Lono on Old Style III https://lonomusic.com/product/old-style-iii-digital-download/
13:05 — Kila Kila Na Roughrider by Lono on Old Style II https://lonomusic.com/product/old-style-ii-digital-download/
15:38 — Moloka’i T.M.H. by Lono on Old Style IV https://lonomusic.com/product/old-style-iv-digital-download/
18:58 — Saturday Market by Lono on Old Style V https://lonomusic.com/product/old-style-v-digital-download/
23:18 — Botonui by Lono on Old Style III https://lonomusic.com/product/old-style-iii-digital-download/
25:23 -- Pala'au State Park parking lot

Support PhotoLukeHawaii:
— PhotoLukeHawaii on YouTube https://www.youtube.com/photolukehawaii
— PhotoLukeHawaii on Twitch https://www.twitch.tv/photolukehawaii
— Join PhotoLukeHawaii Membership on YouTube for $1.99/month to get access to perks, emotes and a badge https://www.youtube.com/channel/UCAUdpc0sN1CewnyOyqQHXcQ/join
— PhotoLukeHawaii on Patreon for only $3 https://www.patreon.com/photolukehawaii
— PhotoLukeHawaii on PayPal https://www.paypal.me/photolukehawaii
— PhotoLukeHawaii merchandise on Teespring at https://www.teespring.com/stores/photolukehawaii
— PhotoLukeHawaii Travels and Bon Dance: https://www.youtube.com/photolukehawaiitravelsandbondance
Follow us on:
— PhotoLukeHawaii Instagram: https://www.instagram.com/photolukehawaii
— PhotoLukeHawaii Facebook: https://www.facebook.com/Photolukehawaii

For all our Hawaii Videos click here https://www.youtube.com/playlist?list=PLxfXsQszwAJQU5bT5MgrQxhEVuZXjIEAU

— Hawaii Covid Information for all islands https://hawaiicovid19.com

PhotoLukeHawaii is an Amazon Affiliate. These links will take you to Amazon.com to the equipment we use to learn more about them or to buy them.  If you buy it through our link we will get credit for it.  Thank you for your support if you do make a purchase!
Equipment Used:
Final Cut Pro X, Adobe Photoshop, Google Earth, Epidemic Sound
iMac 5K 27”, 3 GHz 6-Core Intel Core i5, 32gb Ram, Radeon Pro 570X 4 GB
GoPro Hero 11: https://gopro.com
GoPro Hero 9: https://amzn.to/2FADMDI
GoPro Volta: https://amzn.to/42iJfE3
Hohem Pro 4: https://amzn.to/3nu8QqQ
Magnetic Swivel Clip: https://amzn.to/3ZY63bv
Head Strap w/ Quick Clip: https://amzn.to/3KFKcQY
Mini Handle Tripod: https://amzn.to/3yP41hL
GoPro Suction Cup Mount:  https://amzn.to/2SfMMPx
Micro SD card: https://amzn.to/31aXRI8
GoPro Hero 9 Battery: https://amzn.to/34XtWVG
GoPro Enduro Battery: https://gopro.com
Hero 9, 10, 11 Dual Battery Charger: https://amzn.to/2MksVRb" -metadata "purl=https://www.youtube.com/watch?v=3CUDp_kBB5w" -metadata "comment=https://www.youtube.com/watch?v=3CUDp_kBB5w" -metadata "artist=PhotoLukeHawaii" -movflags "+faststart" "file:Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.temp.mp3"
[ThumbnailsConvertor] Converting thumbnail "Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.webp" to png
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -f image2 -pattern_type none -i "file:Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.webp" -update 1 -movflags "+faststart" "file:Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.png"
[EmbedThumbnail] ffmpeg: Adding thumbnail to "Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.mp3"
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -i "file:Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.mp3" -i "file:Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.png" -c copy -map "0:0" -map "1:0" -write_id3v1 1 -id3v2_version 3 "-metadata:s:v" "title=\"Album cover\"" "-metadata:s:v" "comment=Cover (front)" -movflags "+faststart" "file:Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.temp.mp3"
[SplitChapters] Splitting video by chapters; 10 chapters found
[SplitChapters] Chapter 001; Destination: Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa - 001 Big Wind Kite Factory in Maunaloa [3CUDp_kBB5w].mp3
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -ss 0.0 -t 22.0 -i "file:Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.mp3" -map 0 -dn -ignore_unknown -c copy -movflags "+faststart" "file:Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa - 001 Big Wind Kite Factory in Maunaloa [3CUDp_kBB5w].mp3"
[SplitChapters] Chapter 002; Destination: Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa - 002 Mauka Kamiano by Lono on Molokai IWI [3CUDp_kBB5w].mp3
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -ss 22.0 -t 191.0 -i "file:Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.mp3" -map 0 -dn -ignore_unknown -c copy -movflags "+faststart" "file:Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa - 002 Mauka Kamiano by Lono on Molokai IWI [3CUDp_kBB5w].mp3"
[debug] ffmpeg version n4.3.2-161-g4383b9e8a3 Copyright (c) 2000-2021 the FFmpeg developers
  built with gcc 9.3-win32 (GCC) 20200320
  configuration: --prefix=/ffbuild/prefix --pkg-config-flags=--static --pkg-config=pkg-config --cross-prefix=x86_64-w64-mingw32- --arch=x86_64 --target-os=mingw32 --enable-gpl --enable-version3 --disable-debug --disable-w32threads --enable-pthreads --enable-iconv --enable-zlib --enable-libxml2 --enable-libfreetype --enable-libfribidi --enable-gmp --enable-lzma --enable-fontconfig --enable-opencl --enable-libvmaf --disable-vulkan --enable-libvorbis --enable-amf --enable-libaom --enable-avisynth --enable-libdav1d --enable-libdavs2 --enable-ffnvcodec --enable-cuda-llvm --disable-libglslang --enable-libgme --enable-libass --enable-libbluray --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvpx --enable-libwebp --enable-lv2 --enable-libmfx --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-librav1e --enable-librubberband --enable-schannel --enable-sdl2 --enable-libsoxr --enable-libsrt --enable-libtwolame --enable-libvidstab --enable-libx264 --enable-libx265 --enable-libxavs2 --enable-libxvid --enable-libzimg --extra-cflags=-DLIBTWOLAME_STATIC --extra-cxxflags= --extra-ldflags=-pthread --extra-libs=-lgomp
  libavutil      56. 51.100 / 56. 51.100
  libavcodec     58. 91.100 / 58. 91.100
  libavformat    58. 45.100 / 58. 45.100
  libavdevice    58. 10.100 / 58. 10.100
  libavfilter     7. 85.100 /  7. 85.100
  libswscale      5.  7.100 /  5.  7.100
  libswresample   3.  7.100 /  3.  7.100
  libpostproc    55.  7.100 / 55.  7.100
Input #0, mp3, from 'file:Y:\Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa.mp3':
  Metadata:
    title           : Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa
    encoder         : Lavf58.45.100
    description     : Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa July 29, 2023 Molokai Hawaii. -- Lono Music - https://www.lonomusic.com /  https://www.youtube.com/kaupeepee / MOLOKAI IWI BAND https://www.youtube.com/@molokaiiw
                    :
                    : Menu:
                    : 0:00 -- Big Wind Kite Factory in Maunaloa
                    : 0:22 — Mauka Kamiano by Lono on Molokai IWI https://lonomusic.com/product/iwi-digital-download/
                    : 3:33 — Mauna Alani by Lono on Old Style II https://lonomusic.com/product/old-style-ii-digital-download/
                    : 6:35 — Sunshine Slack by Lono Old Style IV https://lonomusic.com/product/old-style-iv-digital-download/
                    : 10:12 — E Hele Pohaku by Lono on Old Style III https://lonomusic.com/product/old-style-iii-digital-download/
                    : 13:05 — Kila Kila Na Roughrider by Lono on Old Style II https://lonomusic.com/product/old-style-ii-digital-download/
                    : 15:38 — Moloka’i T.M.H. by Lono on Old Style IV https://lonomusic.com/product/old-style-iv-digital-download/
                    : 18:58 — Saturday Market by Lono on Old Style V https://lonomusic.com/product/old-style-v-digital-download/
                    : 23:18 — Botonui by Lono on Old Style III https://lonomusic.com/product/old-style-iii-digital-download/
                    : 25:23 -- Pala'au State Park parking lot
                    :
                    : Support PhotoLukeHawaii:
                    : — PhotoLukeHawaii on YouTube https://www.youtube.com/photolukehawaii
                    : — PhotoLukeHawaii on Twitch https://www.twitch.tv/photolukehawaii
                    : — Join PhotoLukeHawaii Membership on YouTube for $1.99/month to get access to perks, emotes and a badge https://www.youtube.com/channel/UCAUdpc0sN1CewnyOyqQHXcQ/join
                    : — PhotoLukeHawaii on Patreon for only $3 https://www.patreon.com/photolukehawaii
                    : — PhotoLukeHawaii on PayPal https://www.paypal.me/photolukehawaii
                    : — PhotoLukeHawaii merchandise on Teespring at https://www.teespring.com/stores/photolukehawaii
                    : — PhotoLukeHawaii Travels and Bon Dance: https://www.youtube.com/photolukehawaiitravelsandbondance
                    : Follow us on:
                    : — PhotoLukeHawaii Instagram: https://www.instagram.com/photolukehawaii
                    : — PhotoLukeHawaii Facebook: https://www.facebook.com/Photolukehawaii
                    :
                    : For all our Hawaii Videos click here https://www.youtube.com/playlist?list=PLxfXsQszwAJQU5bT5MgrQxhEVuZXjIEAU
                    :
                    : — Hawaii Covid Information for all islands https://hawaiicovid19.com
                    :
                    : PhotoLukeHawaii is an Amazon Affiliate. These links will take you to Amazon.com to the equipment we use to learn more about them or to buy them.  If you buy it through our link we will get credit for it.  Thank you for your support if you do make a purcha
                    : Equipment Used:
                    : Final Cut Pro X, Adobe Photoshop, Google Earth, Epidemic Sound
                    : iMac 5K 27”, 3 GHz 6-Core Intel Core i5, 32gb Ram, Radeon Pro 570X 4 GB
                    : GoPro Hero 11: https://gopro.com
                    : GoPro Hero 9: https://amzn.to/2FADMDI
                    : GoPro Volta: https://amzn.to/42iJfE3
                    : Hohem Pro 4: https://amzn.to/3nu8QqQ
                    : Magnetic Swivel Clip: https://amzn.to/3ZY63bv
                    : Head Strap w/ Quick Clip: https://amzn.to/3KFKcQY
                    : Mini Handle Tripod: https://amzn.to/3yP41hL
                    : GoPro Suction Cup Mount:  https://amzn.to/2SfMMPx
                    : Micro SD card: https://amzn.to/31aXRI8
                    : GoPro Hero 9 Battery: https://amzn.to/34XtWVG
                    : GoPro Enduro Battery: https://gopro.com
                    : Hero 9, 10, 11 Dual Battery Charger: https://amzn.to/2MksVRb
    synopsis        : Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa July 29, 2023 Molokai Hawaii. -- Lono Music - https://www.lonomusic.com /  https://www.youtube.com/kaupeepee / MOLOKAI IWI BAND https://www.youtube.com/@molokaiiw
                    :
                    : Menu:
                    : 0:00 -- Big Wind Kite Factory in Maunaloa
                    : 0:22 — Mauka Kamiano by Lono on Molokai IWI https://lonomusic.com/product/iwi-digital-download/
                    : 3:33 — Mauna Alani by Lono on Old Style II https://lonomusic.com/product/old-style-ii-digital-download/
                    : 6:35 — Sunshine Slack by Lono Old Style IV https://lonomusic.com/product/old-style-iv-digital-download/
                    : 10:12 — E Hele Pohaku by Lono on Old Style III https://lonomusic.com/product/old-style-iii-digital-download/
                    : 13:05 — Kila Kila Na Roughrider by Lono on Old Style II https://lonomusic.com/product/old-style-ii-digital-download/
                    : 15:38 — Moloka’i T.M.H. by Lono on Old Style IV https://lonomusic.com/product/old-style-iv-digital-download/
                    : 18:58 — Saturday Market by Lono on Old Style V https://lonomusic.com/product/old-style-v-digital-download/
                    : 23:18 — Botonui by Lono on Old Style III https://lonomusic.com/product/old-style-iii-digital-download/
                    : 25:23 -- Pala'au State Park parking lot
                    :
                    : Support PhotoLukeHawaii:
                    : — PhotoLukeHawaii on YouTube https://www.youtube.com/photolukehawaii
                    : — PhotoLukeHawaii on Twitch https://www.twitch.tv/photolukehawaii
                    : — Join PhotoLukeHawaii Membership on YouTube for $1.99/month to get access to perks, emotes and a badge https://www.youtube.com/channel/UCAUdpc0sN1CewnyOyqQHXcQ/join
                    : — PhotoLukeHawaii on Patreon for only $3 https://www.patreon.com/photolukehawaii
                    : — PhotoLukeHawaii on PayPal https://www.paypal.me/photolukehawaii
                    : — PhotoLukeHawaii merchandise on Teespring at https://www.teespring.com/stores/photolukehawaii
                    : — PhotoLukeHawaii Travels and Bon Dance: https://www.youtube.com/photolukehawaiitravelsandbondance
                    : Follow us on:
                    : — PhotoLukeHawaii Instagram: https://www.instagram.com/photolukehawaii
                    : — PhotoLukeHawaii Facebook: https://www.facebook.com/Photolukehawaii
                    :
                    : For all our Hawaii Videos click here https://www.youtube.com/playlist?list=PLxfXsQszwAJQU5bT5MgrQxhEVuZXjIEAU
                    :
                    : — Hawaii Covid Information for all islands https://hawaiicovid19.com
                    :
                    : PhotoLukeHawaii is an Amazon Affiliate. These links will take you to Amazon.com to the equipment we use to learn more about them or to buy them.  If you buy it through our link we will get credit for it.  Thank you for your support if you do make a purcha
                    : Equipment Used:
                    : Final Cut Pro X, Adobe Photoshop, Google Earth, Epidemic Sound
                    : iMac 5K 27”, 3 GHz 6-Core Intel Core i5, 32gb Ram, Radeon Pro 570X 4 GB
                    : GoPro Hero 11: https://gopro.com
                    : GoPro Hero 9: https://amzn.to/2FADMDI
                    : GoPro Volta: https://amzn.to/42iJfE3
                    : Hohem Pro 4: https://amzn.to/3nu8QqQ
                    : Magnetic Swivel Clip: https://amzn.to/3ZY63bv
                    : Head Strap w/ Quick Clip: https://amzn.to/3KFKcQY
                    : Mini Handle Tripod: https://amzn.to/3yP41hL
                    : GoPro Suction Cup Mount:  https://amzn.to/2SfMMPx
                    : Micro SD card: https://amzn.to/31aXRI8
                    : GoPro Hero 9 Battery: https://amzn.to/34XtWVG
                    : GoPro Enduro Battery: https://gopro.com
                    : Hero 9, 10, 11 Dual Battery Charger: https://amzn.to/2MksVRb
    purl            : https://www.youtube.com/watch?v=3CUDp_kBB5w
    comment         : https://www.youtube.com/watch?v=3CUDp_kBB5w
    artist          : PhotoLukeHawaii
    date            : 2023
  Duration: 00:26:18.00, start: 0.023021, bitrate: 114 kb/s
    Chapter #0:0: start 0.000000, end 21.931000
    Metadata:
      title           : Big Wind Kite Factory in Maunaloa
    Chapter #0:1: start 21.931000, end 212.931000
    Metadata:
      title           : Mauka Kamiano by Lono on Molokai IWI
    Chapter #0:2: start 212.931000, end 394.931000
    Metadata:
      title           : Mauna Alani by Lono on Old Style II
    Chapter #0:3: start 394.931000, end 611.931000
    Metadata:
      title           : Sunshine Slack by Lono Old Style IV
    Chapter #0:4: start 611.931000, end 784.931000
    Metadata:
      title           : E Hele Pohaku by Lono on Old Style III
    Chapter #0:5: start 784.931000, end 937.931000
    Metadata:
      title           : Kila Kila Na Roughrider by Lono on Old Style II
    Chapter #0:6: start 937.931000, end 1137.931000
    Metadata:
      title           : Moloka’i T.M.H. by Lono on Old Style IV
    Chapter #0:7: start 1137.931000, end 1397.931000
    Metadata:
      title           : Saturday Market by Lono on Old Style V
    Chapter #0:8: start 1397.931000, end 1522.931000
    Metadata:
      title           : Botonui by Lono on Old Style III
    Chapter #0:9: start 1522.931000, end 1577.931000
    Metadata:
      title           : Pala'au State Park parking lot
    Stream #0:0: Audio: mp3, 48000 Hz, stereo, fltp, 107 kb/s
    Metadata:
      encoder         : Lavc58.91
    Stream #0:1: Video: png, rgb24(pc), 1280x720, 90k tbr, 90k tbn, 90k tbc (attached pic)
    Metadata:
      title           : "Album cover"
      comment         : Cover (front)
file:Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa - 002 Mauka Kamiano by Lono on Molokai IWI [3CUDp_kBB5w].mp3: No such file or directory

ERROR: Postprocessing: file:Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa - 002 Mauka Kamiano by Lono on Molokai IWI [3CUDp_kBB5w].mp3: No such file or directory
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 3439, in process_info
  File "yt_dlp\YoutubeDL.py", line 3621, in post_process
  File "yt_dlp\YoutubeDL.py", line 3603, in run_all_pps
  File "yt_dlp\YoutubeDL.py", line 3581, in run_pp
  File "yt_dlp\postprocessor\common.py", line 24, in run
  File "yt_dlp\postprocessor\common.py", line 129, in wrapper
  File "yt_dlp\postprocessor\ffmpeg.py", line 1064, in run
  File "yt_dlp\postprocessor\ffmpeg.py", line 367, in real_run_ffmpeg
yt_dlp.postprocessor.ffmpeg.FFmpegPostProcessorError: file:Hawaii Drive to Phallic Rock and Kalaupapa Lookout Kalae Pala'au State Park Ka Ule O Nanahoa - 002 Mauka Kamiano by Lono on Molokai IWI [3CUDp_kBB5w].mp3: No such file or directory
```

**Comment by bashonly** [comment]
Is `Y:\` a network drive?

It looks like the 2nd chapter's real filepath may be too long for Windows to handle, thus the error.

Try adding `-o "chapter:%(section_number).02d %(section_title).50B.%(ext)s"` to your command to see if that eliminates the error

**Comment by s3n0** [comment]
Hi @bashonly .

Disk Y:\ is a RAM-disk created with ImDisk tool. BTW, I am using Windows 10 and this RAM disk works great in it.

**I updated FFMPEG / FFPROBE / FFPLAY to the version 2023-08 and everything works now.**

I was using the version from 2021 before. Unfortunately, I don't know the exact version, but according to the note (date trace) in the file name, it was marked as 2021.

Unfortunately, I hastily deleted the old version from my disk. So I can't give a report, on your advice, after changing the argument `-o` / `--output`.

I tried to change the argument with the new FFMPEG as you suggest... and everything works. In this case, the square bracket at the end of the file name is no longer added in the output mp3 filenames.

However, if I use my original yt-dlp command line, then in the file names (at the end of them), the square brackets with the number inside, also appear there. Otherwise, even my original command line (with the new FFMPEG version) works fine.

It didn't occur to me that this could also be an incompatibility between the old version of FFMPEG and the new YT-DLP.

It's strange.

Sorry and thank you very much for your help.

--------------------------

**P.S. :**

Here is my current command line for yt-dlp.exe in Windows 10:

`yt-dlp.exe -vU --paths home:Y:\ --output "chapter:%%(section_number).02d %%(section_title).50B.%%(ext)s" --embed-thumbnail --embed-metadata --split-chapters --extract-audio --audio-format mp3 "%URL%"`

The double `%%` character is only there because the command line is from a Batch script (Windows 10). In the case of Shell in Windows 10, of course, you only need to use a single `%' character.
